_data/blogs.yml

```diff
@@ -191,4 +191,9 @@
 - url: https://saarthakchaturvedi05.wordpress.com/2017/12/24/interview-with-long-term-open-source-contributors/
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
-  author_name: Saarthak Chaturvedi
\ No newline at end of file
+  author_name: Saarthak Chaturvedi
+  
+- url    : https://amantiwari123.wordpress.com/2017/12/27/my-journey-to-google-code-in-gci-2017-18/
```

Comment:
You have left extra spaces between `url` and `:` and similarly other key values. Make sure to follow the correct practice and change it the way other entries are mentioned.